doc/man3/OCSP_resp_find_status.pod

```diff
@@ -120,14 +120,17 @@ OCSP_basic_verify() checks that the basic response message B<bs> is correctly
 signed and that the signer certificate can be validated. It takes B<st> as
 the trusted store and B<certs> as a set of untrusted intermediate certificates.
 The function first tries to find the signer certificate of the response
-in <certs>. It also searches the certificates the responder may have included
+in B<certs>. It also searches the certificates the responder may have included
 in B<bs> unless the B<flags> contain B<OCSP_NOINTERN>.
 It fails if the signer certificate cannot be found.
 Next, the function checks the signature of B<bs> and fails on error
 unless the B<flags> contain B<OCSP_NOSIGS>. Then the function already returns
 success if the B<flags> contain B<OCSP_NOVERIFY> or if the signer certificate
 was found in B<certs> and the B<flags> contain B<OCSP_TRUSTOTHER>.
 Otherwise the function continues by validating the signer certificate.
+If B<flags> contains B<OCSP_PARTIAL_CHAIN>, intermediaries that appear
+in the chain are trusted, just like B<X509_V_FLAG_PARTIAL_CHAIN>
+as described in L<X509_VERIFY_PARAM_set_flags(3)/VERIFICATION FLAGS>.
```

Comment:
This looks very confusing to me. I expect that it says that intermediate CAs in the trust store are trust anchors.

Reply:
The whole paragraph is very poorly written and hard to understand; I didn't want to tackle rewriting it for this PR.

Are you confused that I don't describe the semantics of the new flag, but point to another place?  I really don't want to write something twice, and prefer to point to the definitive source.

Reply:
It says "intermediaries that appear in the chain are trusted", which reads to me any intermediary is trusted, whether it's in the trust store or not, and so you're just going to check the end certificate, which just isn't what this does. Either add a few more words to make it clear, or don't describe it at all.

Reply:
AARgh.  Thanks @kroeckx.  Now I see it.  It should say "intermediates found in the trust store are trusted."  Does that make more sense?

Reply:
Yes.

Reply:
But you've now said it's about intermediates in the response, which doesn't seem to be the case.

Reply:
Intermediates are allowed, and we see them in production, hence this patch.

Reply:
It doesn't just affect the intermediates in the response, it also affects intermediates passed in certs.

Reply:
And looking at the x509_vfy code, it also accepts the signing cert.  I tried to keep implementation details out of the manpage, and reworded it yet again.  If you have specific suggestions, please let me know.  Thanks for the careful reading.

Reply:
We're going to check that `signer` is a certificate we trust. The trusted certs are in `st`, and the untrusted in `untrusted`. `untrusted` is the combination of `certs` and `bs->certs`. The effect of settings partial chain is `st` can contain non-root CAs.

So my suggestion is to just write: If B<flags> contains B<OCSP_PARTIAL_CHAIN>, intermediate CA certificates in B<st> are trust-anchors.

Reply:
Sure.  Just one other question "If B"  markup issue?  Do you mean "st" ?

Reply:
I've put some quotes around it.

Reply:
THANKS!  fixup pushed.